tests/system/src/test/java/test/robot/javafx/scene/TextCharacterIndexTest.java

```diff
@@ -133,8 +134,9 @@ public class TextCharacterIndexTest {
 
     private void mouseClick(double x, double y) {
         Util.runAndWait(() -> {
-            robot.mouseMove((int) (scene.getWindow().getX() + scene.getX() + x),
-                    (int) (scene.getWindow().getY() + scene.getY() + y));
+            Window st = scene.getWindow();
+            robot.mouseMove((int) (st.getX() + scene.getX() + x),
+                    (int) (st.getY() + scene.getY() + y));
```

Comment:
two things:
1. May be `Window w` would be better than `st`
2. Should it be (int) or (int)Math.round(...) (or (int)Math.rint(...) ?)

Reply:
>     1. May be `Window w` would be better than `st`
> 

Yes. It is much better. Updated the code.

 >     2. Should it be (int) or (int)Math.round(...) (or (int)Math.rint(...) ?)

I think (int) is sufficient in this case.